### ISSUE TITLE
Error when the infer_sr_scale is set to other values

### ISSUE THREAD

**Issue by CXNing** [THUDM/Inf-DiT#15]
Does the super-resolution task only support 4x? When I set the infer_sr_scale to 8 or 16,  an error occurs:
"RuntimeError: k must have shape (batch_size, seqlen_k, num_heads_k, head_size_og)"

**Comment by yzy-thu** [comment]
Yes, currently this model supports a maximum of 4x super-resolution. If you need higher multiple, you may need to do iterative super-resolution.
